proxy/src/test/java/com/velocitypowered/proxy/scheduler/VelocitySchedulerTest.java

```diff
@@ -65,4 +67,36 @@ void repeatTaskWorks() throws Exception {
     task.cancel();
   }
 
+  @Test
+  void obtainTasksFromPlugin() throws Exception {
+    VelocityScheduler scheduler = new VelocityScheduler(new FakePluginManager());
```

Comment:
I'm nervous about this test. How do we know it will pass or fail? Would feel more comfortable with a `CountDownLock` and doing an assertion in the test thread.

Reply:
If you mean a `CountDownLatch`, I just implemented it in the last commit

Reply:
This is not acceptable. We should NEVER be asserting inside a task.